identity_service/utils/encoder.py

```diff
@@ -0,0 +1,58 @@
+"""Encoder class"""
+from bcrypt import gensalt, hashpw, checkpw
+
+
+class Encoder:
+    """
+    Encoder class
+
+    Methods
+    _______
```

Comment:
remove underscores and type "------" instead

Reply:
fixed